tla-io/src/main/scala/at/forsyte/apalache/io/quint/QuintIR.scala

```diff
@@ -374,27 +318,7 @@ sealed private[quint] trait QuintType
 
 private[quint] object QuintType {
 
-  implicit val rw: RW[QuintType] = RW.merge(
-      QuintBoolT.rw,
-      QuintIntT.rw,
-      QuintStrT.rw,
-      QuintConstT.rw,
-      QuintVarT.rw,
-      QuintSetT.rw,
-      QuintSeqT.rw,
-      QuintFunT.rw,
-      QuintOperT.rw,
-      QuintTupleT.rw,
-      QuintRecordT.rw,
-      QuintSumT.rw,
-  )
-
-  // NOTE: Contrary to quint values, for quint types, source IDs are optional.
-  // This is because many types are inferred, and not derived from the soruce.
-  //
-  // We therefor default id to -1 in the following, because upickle handles
-  // optional values stupidly and if we specify `id` as `Option[Int]` it will
-  // require the value of the ID field to be a (possibly empty) singleton array.
```

Comment:
Unrelated to the PR, this comment was just outdated (the thing it refers to no longer exists)